### PR TITLE
NamespacedImports

### PR DESCRIPTION
**Pull request by tkawachi**
Motivation:
Sometimes .seconds, .minutes, .hours etc. for Int conflict with implicit
provided by other libraries, for example scala.concurrent.duration package
or specs2 or nscala-time. It's useful if we can use these methods in
conjunction with other library.

Idea:
NamespacedImport doesn't add .seconds etc. to Int directly. Instead, it
adds .cs method to Int, which returns `CS[Int]` (Int tagged with chronoscala
marker trait), and provides `CS[Int] => RichInt` implicit conversion.
Then we can write 10.cs.seconds for chronoscala seconds, and reserve
10.seconds for other implicit.

### CODE REVIEW

File: src/main/scala/jp/ne/opt/chronoscala/NamespacedImplicits.scala [hunk 1]
```diff
@@ -0,0 +1,13 @@
+package jp.ne.opt.chronoscala
+
+import jp.ne.opt.chronoscala.Tag.CS
+
+import scala.language.implicitConversions
+
+trait NamespacedImplicits extends Implicits with NamespacedIntImplicits
+
+trait NamespacedIntImplicits extends IntImplicits {
+  override def richInt(n: Int): RichInt = new RichInt(n)
```
Comment (ocadaruma) [inline src/main/scala/jp/ne/opt/chronoscala/NamespacedImplicits.scala]: I think disabling Int conversion by overriding is little bit tricky.

How about following ?

- `NamespacedIntImplicits` extends nothing.
- `NamespacedImplicits extends NamespacedIntImplicits with DurationImplicits with TimeImplicits with OrderingImplicits`
- `NamespacedImports extends TypeImports with StaticForwarderImports with NamespacedImplicits`

**Comment by ocadaruma** [comment]
@tkawachi Thank you for great PR! Please check my comment.

File: src/main/scala/jp/ne/opt/chronoscala/NamespacedImplicits.scala (hunk 1 above)
Comment (tkawachi) [inline src/main/scala/jp/ne/opt/chronoscala/NamespacedImplicits.scala]: That's ok.

Other things to consider is that should we create name spaced variants for other implicit conversions for `Duration`, `ZonedDateTime` etc. These are rarely conflicted, IMO.

Comment (ocadaruma) [inline src/main/scala/jp/ne/opt/chronoscala/NamespacedImplicits.scala]: I think so, too. 
> These are rarely conflicted, IMO.

**Comment by ocadaruma** [comment]
LGTM

**Pull request merged**